docs/local-dev/cli-commands.md

```diff
@@ -237,6 +237,18 @@ Options:
   -h, --help               display help for command
 ```
 
+These are the available `--select` options supported by the `create` command:
+
+| Create Option     | Description                                             | Example                        | Related                                                              |
+| ----------------- | ------------------------------------------------------- | ------------------------------ | -------------------------------------------------------------------- |
+| plugin            | Used to generate a default frontend plugin              | `packages/app`                 | [Create a Plugin](https://backstage.io/docs/plugins/create-a-plugin) |
```

Comment:
Let's use `@backstage/plugin-catalog` instead of app and `@backstage/plugin-catalog-backend`  instead of `backend` as the current examples are private

Reply:
Just updated this.